proto/atomone/dynamicfee/module/v1/module.proto

```diff
@@ -4,6 +4,8 @@ package atomone.dynamicfee.module.v1;
 
 import "cosmos/app/v1alpha1/module.proto";
 
+option go_package = "github.com/atomone-hub/atomone/x/dynamicfee/types/module";
+
```

Comment:
Should we backport that fix for v3 (where dynamicfee module is expected to be shipped) ?